app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,7 @@
     <string name="error_authentication">Invalid user. Please contact your administrator.</string>
     <string name="error_response_messages">Unable to show your messages, please try again later.</string>
     <string name="retry">Retry</string>
+    <string name="text_description">Description</string>
+    <string name="text_date">Date</string>
+    <string name="image_description">ImageStatus</string>
```

Comment:
Same for this

Reply:
This issue is not yet fixed, please do not mark conversations as resolved until the issue is fixed. In this case, check the comment I made in `values-es/strings.xml`

Reply:
Please address this